src/controllers/AmazonController.js

```diff
@@ -35,15 +35,10 @@ controller = {
   getAlbumArt: function() {
     var img = document.querySelector("#nowPlayingSection .albumImage");
     if (!img) return "";
-    return img.src
+	return img.src && img.src.replace('45_', '500_');
```

Comment:
Good luck, the other extensions that do media keys have trouble with amazon because it doesnt like mouse events or something. Whatever magic you have worked here is the only media key extension i've seen work on amazon.